src/docs/output-targets/custom-elements.md

```diff
@@ -66,8 +87,8 @@ will also automatically recursively define any child components as well.
 This flag defaults to `false` when omitted from a Stencil configuration file.
 
 > Note: At this time, components created not using JSX may not be automatically
-  defined. This is a known limitation of the API and users should be aware of
-  it
+> defined. This is a known limitation of the API and users should be aware of
+> it
 
 ### generateTypeDeclarations
```

Comment:
should we have a note here about how this option's behavior is affected by the setting for `customElementsExportBehavior` ? I think it wouldn't need to be anything more complicated than "When set to generate type declarations, Stencil respects the export behavior selected via `customElementsExportBehavior` and generates type declarations specific to the content of that file" or something along those lines

Reply:
I like that 👍